overturemaps/cli.py

```diff
@@ -27,17 +27,34 @@ def get_writer(output_format, path, schema):
     elif output_format == "geojsonseq":
         writer = GeoJSONSeqWriter(path)
     elif output_format == "geoparquet":
-        # Update the geoparquet metadata to remove the file-level bbox which
-        # will no longer apply to this file. Since we cannot write the field at
-        # the end, just remove it as it's optional. Let the per-row bounding
-        # boxes do all the work.
+        # remove geom extents bbox and add covering spec for spatial filters
         metadata = schema.metadata
+        # extract row metadata
+        row_meta = json.loads(metadata[b"org.apache.spark.sql.parquet.row.metadata"])
+        # look for bbox field
+        bbox_field = [r for r in row_meta["fields"] if r["name"] == "bbox"]
+        # extract geo metadata
         geo = json.loads(metadata[b"geo"])
-        for column in geo["columns"].values():
-            column.pop("bbox")
+        # iter geometry columns and associated metadata
+        # there can be multiple geom columns
+        for geom_col_name, geom_col_vals in geo["columns"].items():
```

Comment:
I think you can remove the 'for' loop and just grab the first geo column. There should only be one (it's Overture data). And if there were more, we wouldn't want to re-use the same bbox column as the covering.